Amazon.IonDotnet/Internals/Binary/SymbolTableReader.cs

```diff
@@ -521,9 +520,18 @@ public bool HasAnnotation(string annotation)
         /// <summary>
         /// Dispose SymbolTableReader.
         /// </summary>
-        public void Dispose()
+        public virtual void Dispose()
```

Comment:
The [pattern](https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose#dispose-and-disposeboolean) mentions this should be non-virtual. Does it matter?

Reply:
Ah, yes you are correct because only Dispose(bool) needs to be virtual